src/Spryker/Zed/EventBehavior/Persistence/EventBehaviorQueryContainer.php

```diff
@@ -112,4 +107,19 @@ public function eventBehaviorTableExists()
             return false;
         }
     }
+
+    /**
+     * @deprecated Will be removed without replacement.
+     *
+     * @throws \Spryker\Zed\EventBehavior\Persistence\Exception\EventBehaviorQueryNotExistsException
+     */
+    protected function checkRequiredClasses(): void
```

Comment:
This is very generic name for the query container. I would prefer smth like `eventBehaviorEntityClassExists`